molgenis-data/src/main/java/org/molgenis/data/meta/MetaDataSearchServiceImpl.java

```diff
@@ -44,7 +44,7 @@ public MetaDataSearchServiceImpl(DataService dataService, MetaDataService metaDa
 		else
 		{
 			// Search in packages
-			Query q = new QueryImpl().search(searchTerm);
+			Query<Entity> q = new QueryImpl<>().search(searchTerm);
 			// for (Entity packageEntity : dataService.findAllAsIterable(PackageMetaData.ENTITY_NAME, q))
```

Comment:
This comment already existed. The class was automatically updated due to a refactoring.